utils/sv_easing_functions.py

```diff
@@ -328,35 +328,46 @@ def BounceEaseInOut(p, settings=defaultBounceSettings):
 
 
 easing_dict = {
+    # LINEAR
```

Comment:
Correct. I only did it assuming that this is fairly new development and not a lot of nodes depend on it yet. The easing.py doesn’t care about the dictionary keys (i thought).. other than perhaps the BGL part you just added? I could have done another remapping of in MixNumbers nodes to match the one in sv_easing without changing the order, but this would have added extra lines of code and I was trying to keep these to a minimum via a logical reordering. Let me know and I can revert the sv_easing changes if you want.